eternalcore-core/src/main/java/com/eternalcode/core/feature/home/placeholder/HomeLimitPlaceholderImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.eternalcode.core.feature.home.placeholder;
+
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import com.eternalcode.core.placeholder.PlaceholderReplacer;
+import org.bukkit.entity.Player;
+
+import java.util.Map;
+
+public class HomeLimitPlaceholderImpl implements PlaceholderReplacer {
+
+    private final PluginConfiguration pluginConfiguration;
+
+    public HomeLimitPlaceholderImpl(PluginConfiguration pluginConfiguration) {
+        this.pluginConfiguration = pluginConfiguration;
+    }
+
+    @Override
+    public String apply(String text, Player targetPlayer) {
+        int allowedHomes = this.pluginConfiguration.homes.maxHomes.entrySet()
+            .stream()
+            .filter(entry -> targetPlayer.hasPermission(entry.getKey()))
+            .map(Map.Entry::getValue)
+            .max(Integer::compare)
+            .orElse(0);
+
+        return String.valueOf(allowedHomes);
```

Comment:
This code is copy part of SetHomeCommand#getMaxAmountOfHomes method. Can you move this method to the HomeManager and then use there?